convene-web/config/routes.rb

```diff
@@ -5,6 +5,6 @@
   resources :workspaces
   constraints BrandedDomain.new(Workspace) do
     root 'workspaces#show'
-    get "/:id", to: 'rooms#show'
+    get "/:id", to: 'rooms#show', as: :room
```

Comment:
I think that we want room names on branded domains to purely be an entrance point, so we don't want to overload the `room_path` and instead we want to continue to use `workspace_room_path(workspace.slug, room.slug)` when creating links.

Reply:
Just realized we want to keep the `host/workspaces/:workspaces_slug/rooms/:room_slug`, now I understand this, will make change.